wintun-testing/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+die() {
+  echo "Error: $1"
+  exit 1
+}
+
+[ -z "$TUNNELER_SDK_DIR" ] && die 'set TUNNELER_SDK_DIR'
+
+if [[ $(type -P cmake) ]]; then
+  CMAKE=cmake
+elif [ -x "/c/Program Files/Cmake/bin/cmake.exe" ]; then
```

Comment:
where/s `/c` coming from? cygwin? i don't think it's in stock wsl ?

Reply:
git bash